src/server/process_request.c

```diff
@@ -596,7 +597,9 @@ process_request(int sfds)
 #endif
 
 	/* is the request from a host acceptable to the server */
-	if (access_by_krb == 0 && get_sattr_long(SVR_ATR_acl_host_enable)) {
+	if (request->rq_type != PBS_BATCH_Connect &&
```

Comment:
If Kerberos is enabled, then this will not get verified again for any any request type. So how about not mentioning "request->rq_type != PBS_BATCH_Connect " ?